eva_submission/eload_preparation.py

```diff
@@ -62,20 +67,25 @@ def detect_all(self, taxid=None, reference_accession=None):
             self.debug('Config will be reset')
             self.eload_cfg.backup()
             self.eload_cfg.clear()
+
+        self.convert
         self.detect_submitted_metadata()
         if taxid or reference_accession:
             self.replace_values_in_metadata(taxid=taxid, reference_accession=reference_accession)
         self.check_submitted_filenames()
         self.detect_metadata_attributes()
         self.find_genome()
 
-    def detect_submitted_metadata(self):
+    def find_submitted_metadata(self):
```

Comment:
Could also call `detect_submitted_metadata` before you convert, then you can just use the value in the config and not have to call this method twice.